plugins/powerd/fu-plugin-powerd.c

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2021 Twain Byrnes <binarynewts@google.com>
+ * Copyright (C) 2021 George Popoola <gpopoola@google.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <fwupdplugin.h>
+
+void
+fu_plugin_init (FuPlugin *plugin)
+{
+        fu_plugin_set_build_hash (plugin, FU_BUILD_HASH);
+}
+
+gboolean
+fu_plugin_startup (FuPlugin *plugin, GError **error)
+{
+        return TRUE;
```

Comment:
in my mind this plugin should by default return FALSE and set error until there is something that it can do, or there should be more meat to it before merging anything.

Reply:
Yeah, agreed.

@g2pop, can you go ahead and make that change?

Reply:
Just made the change.

Reply:
If you're returning FALSE then you need to set the `GError` otherwise the plugin loader gets very upset. You need something like this before the return FALSE:

	g_set_error (error,
		     G_IO_ERROR,
		     G_IO_ERROR_NOT_SUPPORTED,
		     "not supported yet");